scripts/check-secrets.sh

```diff
@@ -1,2 +1,10 @@
+EXCLUDE_DIRS=(
+    "core/db/sql"
+    "core/infra/dev_config"
+)
+
+EXCLUDE_PATTERN=$(printf "|^%s" "${EXCLUDE_DIRS[@]}")
+EXCLUDE_PATTERN=${EXCLUDE_PATTERN:1}
+
 # scan the staged files
-git secrets --scan --cached $(git diff --cached --name-only)
+git diff --cached --name-only | grep -Ev "$EXCLUDE_PATTERN" | xargs git secrets --scan --cached
```

Comment:
for some reason the `rowid` tripped this so I had to build in some exclusions, not too stoked about that